Patch by Robert Scheck <robert@fedoraproject.org> for zarafa >= 6.40.4, which removes the
failing installation command of zarafa-archiver (which is not-yet available anyway).

--- zarafa-6.40.4/src/installer/linux/Makefile.am		2010-12-09 10:22:37.000000000 +0100
+++ zarafa-6.40.4/src/installer/linux/Makefile.am.archiver	2010-12-12 01:26:12.000000000 +0100
@@ -108,5 +108,3 @@
 	fi
 	install -d -m 755 ${DESTDIR}/${datarootdir}/doc/zarafa-indexer/example-config
 	mv ${DESTDIR}/${data_configdir}/indexer.cfg ${DESTDIR}/${datarootdir}/doc/zarafa-indexer/example-config
-	install -d -m 755 ${DESTDIR}/${datarootdir}/doc/zarafa-archiver/example-config
-	mv ${DESTDIR}/${data_configdir}/archiver.cfg ${DESTDIR}/${datarootdir}/doc/zarafa-archiver/example-config
--- zarafa-6.40.4/src/installer/linux/Makefile.in		2010-12-09 11:07:03.000000000 +0100
+++ zarafa-6.40.4/src/installer/linux/Makefile.in.archiver	2010-12-12 01:27:00.000000000 +0100
@@ -686,8 +686,6 @@
 	fi
 	install -d -m 755 ${DESTDIR}/${datarootdir}/doc/zarafa-indexer/example-config
 	mv ${DESTDIR}/${data_configdir}/indexer.cfg ${DESTDIR}/${datarootdir}/doc/zarafa-indexer/example-config
-	install -d -m 755 ${DESTDIR}/${datarootdir}/doc/zarafa-archiver/example-config
-	mv ${DESTDIR}/${data_configdir}/archiver.cfg ${DESTDIR}/${datarootdir}/doc/zarafa-archiver/example-config
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
